packages/paste-core/components/callout/package.json

```diff
@@ -25,13 +25,17 @@
     "tsc": "tsc"
   },
   "peerDependencies": {
+    "@twilio-paste/anchor": "^12.0.0",
```

Comment:
All of these are peer dependencies for including Button to Callout (for dismiss functionality).